src/index.html

```diff
@@ -11,7 +11,152 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>        
+          <label for="surname">Surname:</label>
+          <input 
+            type="text" 
+            name="surname" 
+            id="surname" 
+            autocomplete="off"
+            required
+          >
+        </div>
+        <div>
+          <label for="name">Name:</label>
+          <input 
+            type="text" 
+            name="name" 
+            id="name" 
+            autocomplete="off"
+            required
+          >
+        </div>
```

Comment:
```suggestion
          <label for="surname">Surname:</label>
          
          <input 
            type="text" 
            name="surname" 
            id="surname" 
            autocomplete="off"
            required
          >
        </div>
        
        <div>
          <label for="name">Name:</label>
          
          <input 
            type="text" 
            name="name" 
            id="name" 
            autocomplete="off"
            required
          >
        </div>
```

add blank line between multiline components